src/Common/CurrentMetrics.cpp

```diff
@@ -207,6 +207,18 @@
     M(MergeTreeAllRangesAnnouncementsSent, "The current number of announcement being sent in flight from the remote server to the initiator server about the set of data parts (for MergeTree tables). Measured on the remote server side.") \
     M(CreatedTimersInQueryProfiler, "Number of Created thread local timers in QueryProfiler") \
     M(ActiveTimersInQueryProfiler, "Number of Active thread local timers in QueryProfiler") \
+    M(InterfaceHTTPSendBytes, "Bytes sent through HTTP interfaces") \
```

Comment:
Why did you choose CurrentMetric over `ProfileEvents`?
Since you metrics are add-only I see the advantages of using `ProfileEvents`. They are stored in query info.

Reply:
seems to make sense... will try to move it